README.md

```diff
@@ -16,6 +16,31 @@ This plugin supports Magento2 version
 * 2.3.7 and higher
 * 2.4 and higher
 
+## Releases
+
+1. **Major** releases are done ONLY when we absolutely need to. We try to not to introduce breaking changes and do major releases as rare as possible. Average now is **yearly**.
+2. **Minor** releases are scheduled but not limited to **once a month** (first week of a month). 
+3. After each bug fix is resolved, we aim to make a **patch** release. A patch releas is scheduled but not limited to **once in 2 weeks.**
+
+**Note: This can be subject to change based on the load and dependancies of the Integration tools team. **
+
+## Support & Maintenance
+
+There are three levels of support we provide for our Magento 2 plugins.
+1. Level 1 > 12 months  - full support (including new features) *Keep in mind that some new features are not possible on older versions, therefore this is not inclusive of ALL new features that are built.* 
+2. Level 2 > 12 months - 18 months - high priority bugs / security only 
```

Comment:
```suggestion
2. Level 2 > 12 months - 18 months - high priority bugs / security only.
```

Reply:
A formatting suggestion for the level names: 
```suggestion
2. **Level 2 (12 months - 18 months)**: high priority bugs / security only 
```